src/Model/Aggregations/Metric/GeoCentroid.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace AviationCode\Elasticsearch\Model\Aggregations\Metric;
+
+use AviationCode\Elasticsearch\Helpers\HasAttributes;
+
+/**
+ * Class GeoCentroid.
+ *
+ * @property array $value;
+ */
+class GeoCentroid implements \JsonSerializable
+{
+    use HasAttributes;
+
+    /**
+     * GeoCentroid Aggregation constructor.
+     *
+     * @param array $value
+     */
+    public function __construct(array $value)
+    {
+        $this->value = $value['value'];
```

Comment:
This aggregation returns location object and a count
```json
        "<field>": {
            "location": {
                "lat": 51.00982965203002,
                "lon": 3.9662131341174245
            },
            "count": 6
        }
```

We should be able use it simple object

```php
$geoCentroid->location->lat;
$geoCentroid->location->lon;
$geoCentroid->count;
```